### PR TITLE
Fix formatting and adds back interior velocity in tilted bottom boundary layer example

### PR DESCRIPTION
**Pull request by tomchor**
The formatting was off in this example:

image

### CODE REVIEW

**Comment by tomchor** [comment]
I can create a separate issue to address this, but perhaps more importantly, I don't see the interior velocity `V∞` being added anywhere there except in the BCs. Usually we add as a background field to avoid inertial oscillations, and indeed it used to be done like that in [older versions of the docs](https://clima.github.io/OceananigansDocumentation/v0.73.8/generated/tilted_bottom_boundary_layer/),  but at some point this was changed and I don't really understand why.

I think it's okay to run the example without adding it as a background field, but then we're solving the problem in a reference frame that's moving with the interior flow and that should be made explicit and I don't see that explanation there. Personally, I'm in favor of including `V∞` as a background field because it's simpler.

CC @hdrake @glwagner who were in https://github.com/CliMA/Oceananigans.jl/pull/3581

**Review by glwagner — APPROVED** [review]

**Comment by glwagner** [comment]
> I can create a separate issue to address this, but perhaps more importantly, I don't see the interior velocity `V∞` being added anywhere there except in the BCs. Usually we add it as a background field to avoid inertial oscillations, and indeed it used to be done like that in [older versions of the docs](https://clima.github.io/OceananigansDocumentation/v0.73.8/generated/tilted_bottom_boundary_layer/), but at some point this was changed and I don't really understand why (nor can I pinpoint where).
> 
> I think it's okay to run the example without adding it as a background field, but then we're solving the problem in a reference frame that's moving with the interior flow and that should be made explicit, and I don't see that explanation there. Personally, I'm in favor of including `V∞` as a background field because it's simpler.
> 
> CC @hdrake @glwagner who were in #3581

What's the problem with adding it back?

**Comment by tomchor** [comment]
> > I can create a separate issue to address this, but perhaps more importantly, I don't see the interior velocity `V∞` being added anywhere there except in the BCs. Usually we add it as a background field to avoid inertial oscillations, and indeed it used to be done like that in [older versions of the docs](https://clima.github.io/OceananigansDocumentation/v0.73.8/generated/tilted_bottom_boundary_layer/), but at some point this was changed and I don't really understand why (nor can I pinpoint where).
> > I think it's okay to run the example without adding it as a background field, but then we're solving the problem in a reference frame that's moving with the interior flow and that should be made explicit, and I don't see that explanation there. Personally, I'm in favor of including `V∞` as a background field because it's simpler.
> > CC @hdrake @glwagner who were in #3581
> 
> What's the problem with adding it back?

No problem at all, and in my opinion that's what we should do. I'm just checking with the community first since it might have been removed for some reason I'm not aware of. Are you okay if I add it back?

**Comment by glwagner** [comment]
Definitely ok. The problem makes more physical sense, plus it illustrates using a velocity in `BackgroundFields`. Also given that the drag is computed with this now-phantom velocity, the example doesn't make sense right now...

File: examples/tilted_bottom_boundary_layer.jl [hunk 1]
```diff
@@ -134,6 +132,14 @@ drag_bc_v = FluxBoundaryCondition(drag_v, field_dependencies=(:u, :v), parameter
 u_bcs = FieldBoundaryConditions(bottom = drag_bc_u)
 v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
 
+# Note that, similar to the buoyancy boundary conditions, we had to
+# include the background flow in the drag calculation.
+#
+# Let us also create `BackgroundField` for the along-slope interior velocity:
+
+@inline constant_velocity(x, z, t, p) = p.V∞
+V∞_field = BackgroundField(constant_velocity, parameters=(; V∞))
```
Comment (glwagner) [inline examples/tilted_bottom_boundary_layer.jl:141]:
```suggestion
V∞_field = BackgroundField(ConstantField(V∞))
```

It's also easy to make `BackgroundField(V∞)` work. I'm surprised we haven't added that!

Comment (glwagner) [reply examples/tilted_bottom_boundary_layer.jl:141]: It's just a matter of adding

```julia
regularize_background_field(LX, LY, LZ, c::Number, grid, clock) = ConstantField(c)
```

https://github.com/CliMA/Oceananigans.jl/blob/main/src/Fields/background_fields.jl#L63

then you can write

```julia
V∞_field = BackgroundField(V∞)
```

Comment (glwagner) [reply examples/tilted_bottom_boundary_layer.jl:141]: Definitely don't want to encourage the rather convoluted approach of writing a function to return a constant! We must be a force for good

Comment (tomchor) [reply examples/tilted_bottom_boundary_layer.jl]: > It's also easy to make BackgroundField(V∞) work. I'm surprised we haven't added that!

Yeah I added that at first and was also surprised that there was no method for that.

I think I'll add it in another single-purpose PR and then do it more cleanly here :)

Comment (glwagner) [reply examples/tilted_bottom_boundary_layer.jl]: I can add it if you want, it'll be less work

Comment (glwagner) [reply examples/tilted_bottom_boundary_layer.jl]: The CI takes annoyingly long to finish so its nice to combine steps like this I think

Comment (tomchor) [reply examples/tilted_bottom_boundary_layer.jl]: Ah, sorry I may have jumped the gun!

Comment (glwagner) [reply examples/tilted_bottom_boundary_layer.jl]: no worries.

I usually feel swamped so try to do things like this quick. But it's good to take care.

**Pull request merged**